File: app/src/main/java/com/afterroot/photos/fragment/SettingsFragment.java
package com.afterroot.photos.fragment;

import android.os.Bundle;
import android.preference.PreferenceFragment;
import android.support.annotation.Nullable;

import com.afterroot.photos.R;

/**
 * Created by Sandip on 10-04-2017.
 */

public class SettingsFragment extends PreferenceFragment {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences_settings);
    }
}

File: app/src/main/java/com/afterroot/tagit/fragment/SettingsFragment.kt
package com.afterroot.tagit.fragment

import android.os.Bundle
import android.preference.PreferenceFragment
import androidx.preference.PreferenceFragmentCompat

import com.afterroot.tagit.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.preferences_settings)
    }
}
